src/storage/mod.rs

```diff
@@ -1847,47 +1847,34 @@ impl<E: Engine, L: LockManager, F: KvFormat> Storage<E, L, F> {
         }
     }
 
-    fn get_causal_ts(ts_provider: &Option<Arc<CausalTsProviderImpl>>) -> Result<Option<TimeStamp>> {
-        if let Some(p) = ts_provider {
-            match p.get_ts() {
-                Ok(ts) => Ok(Some(ts)),
-                Err(e) => Err(box_err!("Fail to get ts: {}", e)),
-            }
-        } else {
-            Ok(None)
-        }
-    }
+    async fn check_causal_ts_synced(ctx: &mut Context, api_version: ApiVersion) -> Result<()> {
```

Comment:
This is an `async` func, but the name is `synced`?

Reply:
I get the reason, this func is check `the casual is synced or not`. Is it? It's a liittle confused.

Reply:
Yes, it is used for checking `causal-ts-provider` is synced. Ok, I try to rename it.

Reply:
I renamed `check_causal_ts_synced` to `check_causal_ts_flushed`.What do you think?

Reply:
I‘m ok with it.